lib/chatbot-api/functions/api-handler/routes/kendra.py

```diff
@@ -13,28 +13,28 @@ class KendraDataSynchRequest(BaseModel):
     workspaceId: str
 
 
-@router.get("/rag/engines/kendra/indexes")
+@router.resolver(field_name="listKendraIndexes")
 @tracer.capture_method
 def kendra_indexes():
     indexes = genai_core.kendra.get_kendra_indexes()
 
-    return {"ok": True, "data": indexes}
+    return indexes
 
 
-@router.post("/rag/engines/kendra/data-sync")
+@router.resolver(field_name="startKendraDataSync")
 @tracer.capture_method
 def kendra_data_sync():
     data: dict = router.current_event.json_body
     request = KendraDataSynchRequest(**data)
 
     genai_core.kendra.start_kendra_data_sync(workspace_id=request.workspaceId)
 
```

Comment:
There is a bug here that prevents kendra indexes from being sync'd.
You will get the following error: `kendra_data_sync() got an unexpected keyword argument 'workspaceId'`

I've tested the fix below and it resolves the issue.

```suggestion
def kendra_data_sync(workspaceId: str):
    genai_core.kendra.start_kendra_data_sync(workspace_id=workspaceId) 

```

Reply:
Thanks for reporting this.